### ISSUE TITLE
cannot read property 'useCLS' of undefined

### ISSUE THREAD

**Issue by PepperAddict** [alirizwan/sequelize-graphql-schema#19]
I was really hoping this would work, but I'm having some problems...

The error: 
```
  Models.Sequelize.useCLS(sequelizeNamespace);
                   ^

TypeError: Cannot read property 'useCLS' of undefined
    at generateSchema (C:\Users\Funshine\Desktop\webjav\socialpirates\backend\
node_modules\sequelize-graphql-schema\dist\sequelize-graphql-schema.js:870:20)
    at Function.module.exports (C:\Users\Funshine\Desktop\webjav\socialpirates
\backend\src\middleware\index.js:50:31)
    at Function.configure (C:\Users\Funshine\Desktop\webjav\socialpirates\back
end\node_modules\@feathersjs\feathers\lib\application.js:59:8)
    at Object.<anonymous> (C:\Users\Funshine\Desktop\webjav\socialpirates\back
end\src\app.js:61:5)
    at Module._compile (internal/modules/cjs/loader.js:799:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:810:10)
    at Module.load (internal/modules/cjs/loader.js:666:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:606:12)
    at Function.Module._load (internal/modules/cjs/loader.js:598:3)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.<anonymous> (C:\Users\Funshine\Desktop\webjav\socialpirates\back
end\src\index.js:3:13)
    at Module._compile (internal/modules/cjs/loader.js:799:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:810:10)
    at Module.load (internal/modules/cjs/loader.js:666:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:606:12)
    at Function.Module._load (internal/modules/cjs/loader.js:598:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:862:12)
    at internal/main/run_main_module.js:21:11
```

my middleware where I'm sticking it in: 
```
  const { GraphQLSchema } = require('graphql');
  const graphqlHTTP = require('express-graphql');

  var options = {
    exclude: ["Users"]
  }

  const {generateSchema} = require('sequelize-graphql-schema')(options);
const models = require( process.cwd() + '\\src\\models\\activity.js');
app.use(
  '/graphql',
  graphqlHTTP({
    schema: new GraphQLSchema(generateSchema(models)),
    graphiql: true
  })
)
```

One example of models named activity.js
```
module.exports = function(sequelize, DataTypes) {
	return sequelize.define('activity', {
		activityID: {
			type: DataTypes.INTEGER,
			allowNull: false,
			primaryKey: true,
			autoIncrement: true
		},
		belongsTo: {
			type: DataTypes.INTEGER,
			allowNull: false,
			references: {
				model: 'users',
				key: 'id'
			},
			unique: true
		},
		profileID: {
			type: DataTypes.INTEGER,
			allowNull: true,
			references: {
				model: 'profile',
				key: 'profileID'
			}
		}
	}, {
		tableName: 'activity'
	});
};

```

**Comment by alirizwan** [comment]
Hi @PepperAddict can you please paste how you are passing sequelize objects to the library. I think sequelize object passing is not much clear in the documentation. It expects in a way which not many knows about. If you could show me how you are setting up your models I can help you fix this. For example you can take a look at https://github.com/alirizwan/sequelize-graphql-schema/blob/develop/tests/models/index.js

Meanwhile I will work on some documentation to make it more clear.

**Comment by PepperAddict** [comment]
Thank you @alirizwan for getting back to me so quickly! 
I actually don't have a bundled index.js version of my models. I'll give that a try later today and get back to you.

**Comment by alirizwan** [comment]
i am making changes today which would let you pass Sequelize as a parameter instead of bundling it with models.

**Comment by PepperAddict** [comment]
ah okay so what you have is completely different from mine. I use feathersjs express so this just makes the transition pretty confusing. 
It doesn't have an indexjs in the models folder, but all the models goes to a sequelize.js file outside of it that bundles all the models. 

It's written like this in a module.exports function:
```
  const sequelize = new Sequelize(connectionString, {
    dialect: 'postgres',
    dialectOptions: {
      ssl: true
    },
    logging: false,
    operatorsAliases: false,
    define: {
      freezeTableName: true
    }
  });
  const oldSetup = app.setup;
```
```
  app.set('sequelizeClient', sequelize);

  app.setup = function (...args) {
    const result = oldSetup.apply(this, args);

    // Set up data relationships
    const models = sequelize.models;
    Object.keys(models).forEach(name => {
      if ('associate' in models[name]) {
        models[name].associate(models);
      }
    });

    // Sync to the database
    sequelize.sync();

    return result;
  };
```

I tried to set it up your way, but I am confused by the config part and couldn't get it to work.

**Comment by alirizwan** [comment]
it's alright. It's better to make an option to pass it as an argument instead of current way.

**Comment by alirizwan** [comment]
@PepperAddict please use 0.1.66

**Comment by PepperAddict** [comment]
hi @alirizwan ! It doesn't give 0.1.66 as a version option yet. 
What goes in the third argument? req? 
And thank you for helping me with this.

**Comment by alirizwan** [comment]
Third argument is req in case of express.

Sent from my iPhone

On Apr 23, 2019, at 1:21 AM, Jenearly Reassey <notifications@github.com<mailto:notifications@github.com>> wrote:


hi @alirizwan<https://github.com/alirizwan> ! It doesn't give 0.1.66 as a version option yet.
What goes in the third argument? req?
And thank you for helping me with this.

**Comment by alirizwan** [comment]
I can confirm 0.1.66 is published on npm.

Sent from my iPhone

On Apr 23, 2019, at 1:21 AM, Jenearly Reassey <notifications@github.com<mailto:notifications@github.com>> wrote:


hi @alirizwan<https://github.com/alirizwan> ! It doesn't give 0.1.66 as a version option yet.
What goes in the third argument? req?
And thank you for helping me with this.

**Comment by PepperAddict** [comment]
I checked npm and you're right, i see a version 0.1.66  , but unfortunately I can't seem to pull that version. I thought maybe I had to wait a few hours and try again in the morning, but unfortunately I get the same thing: 

![Screenshot (74)](https://user-images.githubusercontent.com/4171505/56585599-28838900-65ac-11e9-896d-7b5f67f64d1f.png)

Is there something I'm supposed to do to get the latest version? I tried `yarn add sequelize-graphql-schema@0.1.66` and I've tried using npm and I get the same thing.
